### PR TITLE
fix: LR scheduler's final LR in TF scripts

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR updates the lr scheduler in tf scripts to have the same training parameters in both tensorflow and pytorch scripts. The changes were already made locally on the training machine before launching trainings, nothing to worry about.

### CODE REVIEW

**Review by fg-mindee — APPROVED** [review]
Thanks!
